_templates/footer.php

```diff
@@ -38,6 +38,10 @@
                 <li><a href="<?php echo $sitewide['root'].'open-source'; ?>">Open Source</a></li>
             </ul>
         </footer>
+        <div id="js-alert">
+            <p><strong>JavaScript is required</strong> for parts of this site, like downloading elementary OS and some interactive components.</p>
+        </div>
+        <script>document.getElementById("js-alert").remove ();</script>
```

Comment:
I'd just put this in a `<noscript>`. Because it's a different script block, even if the regular JS errors out, this will still run, so it's no different than `noscript`.

Reply:
Derp, somehow I always forget about `<noscript>`